portfolio/src/main/webapp/script.js

```diff
@@ -26,7 +26,7 @@ var projectList = [
  */
 function showPicture(picture) {
     const portrait = document.getElementById(picture);
-    const pictureButton = document.getElementById("picture-button")
+    const pictureButton = document.getElementById(picture + "-button");
```

Comment:
it might be better to pass in both pictureId and pictureButtonId instead of relying on the fact that they must all have `picture` in the id. This will make things less brittle.

Reply:
Ooo you're right